test/test_in_toto_sign.py

```diff
@@ -0,0 +1,190 @@
+
+"""
+<Program Name>
+  test_in_toto_sign.py
+<Author>
+  Sachit Malik <i.sachitmalik@gmail.com>
+<Started>
+  Wed Jun 21, 2017
+<Copyright>
+  See LICENSE for licensing information.
+<Purpose>
+  Test in_toto_sign command line tool.
+"""
+
+import os
+import sys
+import unittest
+import logging
+import argparse
+import shutil
+import tempfile
+from mock import patch
+from in_toto.models.layout import Layout
+from in_toto.util import (generate_and_write_rsa_keypair,
+  prompt_import_rsa_key_from_file, import_rsa_key_from_file)
+from in_toto.models.link import Link
+from in_toto.in_toto_sign import main as in_toto_sign_main
+from in_toto.in_toto_sign import add_sign, replace_sign, verify_sign
+from in_toto import log
+from in_toto import exceptions
+
+WORKING_DIR = os.getcwd()
+
+# Suppress all the user feedback that we print using a base logger
+logging.getLogger().setLevel(logging.CRITICAL)
+
+class TestInTotoSignTool(unittest.TestCase):
+  """Test in_toto_sign's main() - requires sys.argv patching; error logs/exits
+  on Exception. """
+
+  @classmethod
+  def setUpClass(self):
+     # Backup original cwd
+    self.working_dir = os.getcwd()
+
+    # Find demo files
+    demo_files = os.path.join(
+        os.path.dirname(os.path.realpath(__file__)), "demo_files")
+
+    # Create and change into temporary directory
+    self.test_dir = os.path.realpath(tempfile.mkdtemp())
+    os.chdir(self.test_dir)
+
+    # Copy demo files to temp dir
+    for file in os.listdir(demo_files):
+      shutil.copy(os.path.join(demo_files, file), self.test_dir)
```

Comment:
IMHO something like "file_path" is better, because `listdir` does not return file objects but paths. ;)